package.json

```diff
@@ -16,8 +16,12 @@
   },
   "typesVersions": {
     "*": {
-      "button": ["dist/button.d.ts"],
-      "tag": ["dist/tag.d.ts"]
+      "button": [
+        "dist/button.d.ts"
+      ],
+      "tag": [
+        "dist/tag.d.ts"
+      ]
```

Comment:
Formatter change. I already turned off prettier. not sure what is causing this.

Reply:
hmm Eslint possibly?